### ISSUE TITLE
[DOC] Enhance ThinRuntime example doc to follow the commonly-used specification style

### ISSUE THREAD

**Issue by TrafalgarZZZ** [fluid-cloudnative/fluid#2517]
**Provide a link to that doc page:**

**What is the defect and your suggestions on improvement:**
The doc now uses the following dataset example to access Minio (set Minio endpoint in `spec.mounts[x].mountPoint` and set bucket in `spec.mounts[x].options`).

```yaml
apiVersion: data.fluid.io/v1alpha1
kind: Dataset
metadata:
  name: minio-demo
spec:
  mounts:
  - mountPoint: http://minio:9000
    name: minio
    options:
      bucket: my-first-bucket
    encryptOptions:
      - name: minio-access-key
        valueFrom:
          secretKeyRef:
            name: minio-secret
            key: minio-access-key
      - name: minio-access-secret
        valueFrom:
          secretKeyRef:
            name: minio-secret
            key: minio-access-secret
```

It may be more user-friendly to set bucket in `spec.mounts[x].mountPoint` with a storage scheme like `minio://` as its prefix. Understanding such example is more easy because it follows the same specification commonly used in other Runtimes like AlluxioRuntime and JuicefsRuntime.

To sum up,  I suggest to modify the example like:
```yaml
apiVersion: data.fluid.io/v1alpha1
kind: Dataset
metadata:
  name: minio-demo
spec:
  mounts:
  - mountPoint: minio://my-first-bucket
    name: minio
    options:
      minio-url: http://minio:9000
    encryptOptions:
      - name: minio-access-key
        valueFrom:
          secretKeyRef:
            name: minio-secret
            key: minio-access-key
      - name: minio-access-secret
        valueFrom:
          secretKeyRef:
            name: minio-secret
            key: minio-access-secret
```
